packages/client/package.json

```diff
@@ -15,8 +15,8 @@
   "scripts": {
     "storybook": "start-storybook -p 6006 -c .storybook",
     "build-storybook": "build-storybook",
-    "lint": "eslint --fix ./**/*.{ts,tsx,js}",
-    "precommit": "lint-staged",
+    "lint": "eslint --fix ./**/*.{ts,tsx}",
+    "precommit": "git add . && lint-staged",
```

Comment:
When I created the merge commit from master, this precommit command failed when run from the root of the monorepo. The output was:

```
~/Source/github/Parabol/action.git ⑂fix-prettier +  git commit
husky > pre-commit (node v12.7.0)
lerna notice cli v3.16.4
lerna info Executing command in 3 packages: "yarn run precommit"
parabol-client: yarn run v1.19.1
parabol-client: $ git add . && lint-staged
parabol-client: fatal: not a git repository: '.git'
parabol-client: error Command failed with exit code 128.
parabol-client: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run precommit exited 128 in 'parabol-client'
husky > pre-commit hook failed (add --no-verify to bypass)
```

I think this is happening because the hook is running from `action/` and not `action/packages/client`. What's the purpose of the `git add .` here?
```